src/rt/util/container/treap.d

```diff
@@ -27,9 +27,10 @@ nothrow:
         removeAll();
     }
 
-    void initialize()
+    void initialize(ulong randSeed)
```

Comment:
Would it be a problem to keep the existing `initialize()` method, but also allow passing the seed explicitly in an overload? I'm not expecting Treap to be used often, but having to come up with a magic number can be annoying.

Reply:
Maybe define `rand()` as the default parameter value?